pkg/volume/flexvolume/api/objects.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+// Probe plugin
+
+type FlexVolumeDriverCapabilities struct {
+	AccessMode          api.PersistentVolumeAccessMode
+	DynamicProvisioning bool
+	Attachment          bool
+	LocalAttachment     bool // When Attachement is true, issue the attach/detach calls from the target host, not the controller.
+	CustomMount         bool // This driver has a custom mount/unmount logic. Mount/Unmount must be called.
```

Comment:
Thanks. I was asking for an example where you think CustomMount should be set to true.
I am arguing that there is no such case, and so the field is not required.